django_heroku/core.py

```diff
@@ -43,6 +43,19 @@ def teardown_databases(self, old_config, **kwargs):
 
 
 def settings(config, *, db_colors=False, databases=True, test_runner=True, staticfiles=True, allowed_hosts=True, logging=True, secret_key=True):
+    if 'USE_MAX_CONN_AGE' in os.environ:
+        use_max_conn_age = bool(os.environ['USE_MAX_CONN_AGE'])
+    else:
+        use_max_conn_age = True
+
+    # DEVELOPMENT mode configuration.
+    ssl_required = True
+    if 'DEVELOPMENT' in os.environ:
+        logger.info('Adding $DEVELOPMENT to DEVELOPMENT Django setting.')
+        # Set the Django setting from the environment variable.
+        config['DEVELOPMENT'] = os.environ['DEVELOPMENT']
+        if os.environ['DEVELOPMENT'] == 'True':
+            ssl_required = False
```

Comment:
What if we have a dictionary, perhaps named `dj_database_config` that aggregates these options. So above we would set

```py
dj_database_config['conn_max_age'] = MAX_CONN_AGE
```

If someone wants to use it and then here we set

```py
dj_database_config['ssl_required'] = True  # or False
```

And then below where we `.parse()` or `.config()` we just have one call, e.g.,


```py
config['DATABASES'][db_color] = dj_database_url.parse(url, **dj_database_config)
```

and

```py
config['DATABASES']['default'] = dj_database_url.config(**dj_database_config)
```

Reply:
@sigmavirus24 @burcaw 

What if this was simply:

`ssl_require = not DEBUG`

> What if we have a dictionary, perhaps named dj_database_config that aggregates these options. So above we would set

I agree this particular setting is counter intuitive and tripped me up. 

More specific settings could still be tweaked by updating `DATABASES['default']` after `django_heroku` does it's thing, but this one woudl be nice to "just work".